components/sections/Navbar/index.tsx

```diff
@@ -0,0 +1,77 @@
+import Image from 'next/image';
+import type { FC, ComponentProps } from 'react';
+
+import LanguageDropdown from '@/components/Common/LanguageDropDown';
+import ThemeToggle from '@/components/Common/ThemeToggle';
+import NavItem from '@/components/sections/NavItem';
+
+import style from './index.module.css';
+
+type NavItem = { text: string; href: string };
+
+type NavbarProps = {
+  navItems: NavItem[];
+  languages: ComponentProps<typeof LanguageDropdown>;
+  onThemeTogglerClick: () => void;
+};
+
+const Navbar: FC<NavbarProps> = ({
+  navItems,
+  languages,
+  onThemeTogglerClick,
+}) => {
+  return (
+    <nav className={style.container}>
+      <div className={style.leftItems}>
+        <Image
+          className="dark:hidden"
+          height={24}
+          width={80}
+          src="/static/images/logos/horizontal-dark.svg"
+          alt="Node.js"
+        />
+        <Image
+          className="hidden dark:block"
+          height={24}
+          width={80}
+          src="/static/images/logos/horizontal-light.svg"
```

Comment:
```suggestion
          src={`${BASE_PATH}/static/images/logos/horizontal-light.svg`}
```

For all these links, important to add the `BASE_PATH` variable from `next.constants.mjs` as sometimes deployments happen in subfolders, not the / of domains.

Reply:
(Applies for all the calls of `/static` here